### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: hands-on-exercise/5-ibc-adv/2-hermes-relayer.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ SUBCOMMANDS:
     clear           Clear objects, such as outstanding packets on a channel
     config          Validate Hermes configuration file
     create          Create objects (client, connection, or channel) on chains
-    health-check    Performs a health check of all chains in the the config
+    health-check    Performs a health check of all chains in the config
```
Comment (xavierlepretre) [inline hands-on-exercise/5-ibc-adv/2-hermes-relayer.md:70]: @murataniloener Is this not a straight copy of the output?

**Review by xavierlepretre — APPROVED** [review]

File: hands-on-exercise/5-ibc-adv/2-hermes-relayer.md (hunk 1 above)
Comment (murataniloener) [reply hands-on-exercise/5-ibc-adv/2-hermes-relayer.md:70]: Yes it is

Comment (xavierlepretre) [reply hands-on-exercise/5-ibc-adv/2-hermes-relayer.md:70]:
```suggestion
    health-check    Performs a health check of all chains in the the config
```

So I'd say we keep it as is, even if it is silly. Thoughts?

Comment (murataniloener) [reply hands-on-exercise/5-ibc-adv/2-hermes-relayer.md:70]: No I would fix it, perhaps it is already fixed in the most recent version of Hermes.

**Review by CitMC — APPROVED** [review]

**Review by AndrewB9lab — APPROVED** [review]
Language and style review finished.

**Review by coldice — APPROVED** [review]

**Pull request merged**